catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorTest.java

```diff
@@ -293,6 +306,86 @@ public void testRouteCreationMissingMonitoredDirectory() throws Exception {
         empty());
   }
 
+  @Test
+  public void testInterruptedWhileWaitingForInputTransformers() throws InterruptedException {
+    CdmServiceWaiter serviceWaiter = mock(CdmServiceWaiter.class);
+    doThrow(new InterruptedException()).when(serviceWaiter).waitFor(anySet());
+
+    CdmState cdmState = new CdmState();
+    ContentDirectoryMonitor monitor =
+        new ContentDirectoryMonitor(
+            camelContext,
+            mock(AttributeRegistry.class),
+            1,
+            1,
+            Runnable::run,
+            serviceWaiter,
+            cdmState);
+    monitor.systemSubjectBinder = exchange -> {};
+    monitor.setNumThreads(1);
+    monitor.setReadLockIntervalMilliseconds(1000);
+
+    submitConfigOptions(monitor, monitoredDirectoryPath, ContentDirectoryMonitor.MOVE);
+    assertThat(
+        "The content directory monitor should have no route definitions",
+        monitor.getRouteDefinitions(),
+        hasSize(0));
+    assertThat(cdmState.isRoutesStarted(), is(false));
+  }
+
+  @Test
+  public void testInitWithConfigDeleted() {
+    CdmServiceWaiter serviceWaiter = mock(CdmServiceWaiter.class);
+
+    CdmState cdmState = new CdmState();
+    cdmState.configDeleted();
+
+    ContentDirectoryMonitor monitor =
+        new ContentDirectoryMonitor(
+            camelContext,
+            mock(AttributeRegistry.class),
+            1,
+            1,
+            Runnable::run,
+            serviceWaiter,
+            cdmState);
+    monitor.systemSubjectBinder = exchange -> {};
+    monitor.setNumThreads(1);
+    monitor.setReadLockIntervalMilliseconds(1000);
+
+    submitConfigOptions(monitor, monitoredDirectoryPath, ContentDirectoryMonitor.MOVE);
+    assertThat(
+        "The content directory monitor should have no route definitions",
+        monitor.getRouteDefinitions(),
+        hasSize(0));
+  }
+
+  @Test
+  public void testDestroyWithNoRoutesStarted() throws Exception {
```

Comment:
Need to update this test so the assertions are more accurate. Could pass for other reasons right now.